fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/character/Character.java

```diff
@@ -0,0 +1,19 @@
+package ch.bbw.fabbwled.lands.character;
+
+import lombok.With;
+import java.util.List;
+public class Character {
+    @With
+    public record CharacterDto(String name,
```

Comment:
there is some overlap/conflicts still with what belongs to the `PlayerDto` and what belongs to `CharacterDto`. For example `name` exists in both objects, but means the same. `description` is only useful lore needed during character creation and not used later while playing the game.
How about the following:
```
public record CharacterCreateDto(PlayerDto player, String description);

public record PlayerDto(String name,
                            SectionId currentSection,
                            Set<String> titlesAndHonours,
                              RankEnum rank,
							  ProfessionEnum profession,
                               int stamina,
                               StatsDto baseStats,
                               List<String> posessions)
```
...it is then up to the FE to display what they want during character creation.